### PR TITLE
More mappings for glibc

### PR DESCRIPTION
**Pull request by alejandro-colomar**

### CODE REVIEW

File: gcc.symbols.imp [hunk 1]
```diff
@@ -43,6 +43,7 @@
   { symbol: [ "fsblkcnt_t", private, "<sys/statvfs.h>", public ] },
   { symbol: [ "fsfilcnt_t", private, "<sys/types.h>", public ] },
   { symbol: [ "fsfilcnt_t", private, "<sys/statvfs.h>", public ] },
+  { symbol: [ "getopt", private, "<unistd.h>", public ] },
```
Comment (kimgr) [inline gcc.symbols.imp:46]: The getopt changes kind-of interfere with a pet project of mine -- https://github.com/kimgr/getopt_port. Someone using that compatibility port on Windows will be recommended `<unistd.h>`, which is not available.

That said, our mappings are already very POSIX-/glibc-oriented, so I guess that's fine for now. We should look into making mappings easier to override.

**Comment by kimgr** [comment]
Thanks! Overall, this looks good. A little comment inline on `getopt` and friends, but I'm pretty sure we don't need to worry about it.

File: gcc.symbols.imp (hunk 1 above)
Comment (alejandro-colomar) [reply gcc.symbols.imp:46]: I guess the same issue would be triggered by libc or cc developers when running iwyu(1) on their code base.  I'd like that to be possible, so yes, there should be some way of telling iwyu(1) that it's running on a special case.

**Comment by alejandro-colomar** [comment]
:-)

**Comment by kimgr** [comment]
Merged, thanks!

**Comment by firewave** [comment]
Looks like this fixed #1001.

**Pull request merged**